tests/e2e/specs/shopper/cart-checkout/checkout.test.js

```diff
@@ -53,6 +53,31 @@ describe( 'Shopper → Checkout', () => {
 			}
 		);
 	} );
+	describe( 'Shopper can change the payment method', () => {
```

Comment:
There is only 1 test in this describe block so I'd suggest removing the `beforeAll` and `afterAll` hooks, and just calling `shopper.block.emptyCart()` at the start of the test.

I'd also suggest keeping to the current `describe` and `it` naming conventions. You can look at the other tests in this file for examples, but essentially the describe block should include the sub section we are testing (in this case "Payment Methods") and the `it` should be of the for "user can....."